modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/idp/mgt/ResidentIDPConfigsTestCase.java

```diff
@@ -0,0 +1,262 @@
+                "https://localhost:9853/oauth2/revoke",
+                "Expected Token Revocation Endpoint URL is not found");
+        Assert.assertEquals(fedAuthConfigMap.get("OIDCLogoutEPUrl"),
+                "https://localhost:9853/oidc/logout",
+                "Expected Logout Endpoint URL not found");
+        Assert.assertEquals(fedAuthConfigMap.get("OAuth2AuthzEPUrl"),
+                "https://localhost:9853/oauth2/authorize",
+                "Expected Authorization Endpoint URL is not found");
+        Assert.assertEquals(fedAuthConfigMap.get("OAuth2DCREPUrl"),
+                "https://localhost:9853/api/identity/oauth2/dcr/v1.1/register",
+                "Expected Dynamic Client Registration Endpoint URL is not found");
+        Assert.assertEquals(fedAuthConfigMap.get("OIDCDiscoveryEPUrl"),
+                "https://localhost:9853/oauth2/oidcdiscovery",
+                "Expected Dynamic Client Registration Endpoint URL is not found");
+        Assert.assertEquals(fedAuthConfigMap.get("OAuth2JWKSPage"),
+                "https://localhost:9853/oauth2/jwks", "Expected is not found");
+        Assert.assertEquals(fedAuthConfigMap.get("OAuth2TokenEPUrl"),
+                "https://localhost:9853/oauth2/token",
+                "Expected Token Endpoint URL is not found");
+        Assert.assertEquals(fedAuthConfigMap.get("OAuth2UserInfoEPUrl"),
+                "https://localhost:9853/oauth2/userinfo",
+                "Expected User Info Endpoint URL is not found");
+
+        //Extract provisioning configurations.
+        ProvisioningConnectorConfig[] provisioningConfigs = idProvider.getProvisioningConnectorConfigs();
+
+        Map<String, String> provisioningConfigMap = new HashMap<String, String>();
+        for (ProvisioningConnectorConfig config : provisioningConfigs) {
+            for (Property property: config.getProvisioningProperties()) {
+                provisioningConfigMap.put(property.getName(), property.getValue());
+            }
+        }
+
+        Assert.assertEquals(provisioningConfigMap.get("scimGroupEndpoint"),
+                "https://localhost:9853/wso2/scim/Groups",
+                "Expected SCIM Group Endpoint is not found");
+        Assert.assertEquals(provisioningConfigMap.get("scimUserEndpoint"),
+                "https://localhost:9853/wso2/scim/Users",
+                "Expected SCIM User Endpoint is not found");
+        Assert.assertEquals(provisioningConfigMap.get("scim2UserEndpoint"),
+                "https://localhost:9853/wso2/scim2/Users",
+                "Expected SCIM 2.0 User Endpoint is not found");
+        Assert.assertEquals(provisioningConfigMap.get("scim2GroupEndpoint"),
+                "https://localhost:9853/wso2/scim2/Groups",
+                "Expected SCIM 2.0 Group Endpoint is not found");
+
+        ConfigurationContext configContext = ConfigurationContextFactory.createConfigurationContextFromFileSystem
```

Comment:
Can we move the tenant case to a new test case?

Reply:
Done in https://github.com/wso2/product-is/pull/8370/commits/f196fdb5ff9850329b87aaffb483f2d68783cb04